client/py/examples/dev/multi_stream.py

```diff
@@ -0,0 +1,67 @@
+import synnax as sy
```

Comment:
When I run this example I get a cesium error, could you double check you get that too?

Reply:
Works fine for me

Reply:
Perfect, works for me too now